eclair-core/src/main/resources/reference.conf

```
@@ -160,6 +160,16 @@ eclair {
     }
   }
 
+  payment {
```

Should we name it after the package, like `relay`?
```suggestion
  send {
```

Let's dig another 🐰 🕳️ !
```
  relay {
    ...
  }

  send {  
    max-payment-attempts = 5
    recipient-final-expiry {
      min-delta = 150 // minimum value to add to the current block height
      max-delta = 350 // maximum value to add to the current block height
    }
  }

  receive {
    multi-part-payment-expiry = 60 seconds // default expiry for receiving all parts of a multi-part payment
    invoice-expiry = 1 hour // default expiry for invoices generated by this node
    purge-expired-invoices {
      enabled = true // enable automatic purges of expired invoices from the database
      interval = 24 hours // interval between expired invoice purges
    }
  }
```

---

Definitely a good idea, I didn't want to move existing values in the new section in this PR but we should clearly do a follow-up PR for that! I renamed the section to `send` in https://github.com/ACINQ/eclair/pull/2469/commits/8c1b462073f90d4e68a6cc15e006a7a085419fc8